style.css

```diff
@@ -0,0 +1,1226 @@
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .cover-story-article header {
+    color: white;
+    padding: 14px 12px 12px 12px;
+    z-index: 2;
+  }
+
+  .cover-story-article h4 {
+    font-family: sans-serif;
+    font-size: 14px;
+    font-weight: 500;
+    text-transform: uppercase;
+    letter-spacing: 0.04em;
+    line-height: 1.5;
+  }
+
+  .cover-story-article h2 {
+    font-family: sans-serif;
+    font-size: 20px;
+    font-weight: 700;
+  }
+
+  .funding-rounds {
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+  }
+
+  .funding-rounds-header {
+    color: #f42;
+    margin-bottom: 24px;
+  }
+
+  .funding-rounds-header h2 {
+    font-style: sans-serif;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+  }
+
+  .funding-rounds-header a {
+    text-decoration: none;
+    color: #f42;
+    line-height: 18px;
+    margin-top: 11px;
+  }
+
+  .funding-rounds a {
+    display: block;
+  }
+
+  .funding-rounds article {
+    min-height: 60px;
+    background: #fffaf8;
+    margin-bottom: 24px;
+  }
+
+  .latest-news {
+    margin-top: 2rem;
+    padding: 1.2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: stretch;
+    align-items: stretch;
+  }
+
+  .section-header {
+    margin-bottom: 1.5rem;
+  }
+
+  .section-header a {
+    color: #f42;
+    font-size: 28px;
+    font-weight: 700;
+  }
+
+  .caret {
+    display: inline-block;
+    height: 16px;
+    width: 16px;
+    background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="none" stroke="%23F42" stroke-width="3" stroke-miterlimit="10" d="M4.75 1.5l6.5 6.5-6.5 6.5"/></svg>');
+    background-size: 16px 16px;
+  }
+
+  .latest-news-article-container {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(8, 1fr);
+    grid-gap: 12px;
+  }
+
+  .latest-news-article {
+    display: grid;
+    grid-template-columns: 75px 1fr;
+    grid-template-rows: 1fr;
+    grid-gap: 8px;
+  }
+
+  .article-image-anchor {
+    display: block;
+    color: inherit;
+    min-height: 72px;
+    min-width: 75px;
+    background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/stress-516x270.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: 50%;
+  }
+
+  .article-details {
+    justify-self: center;
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .article-link a {
+    color: #555;
+    font-size: 13px;
+    font-weight: 400;
+    line-height: 1.5;
+  }
+
+  .article-details h4 {
+    margin-bottom: 8px;
+  }
+
+  .information > a {
+    color: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin-bottom: 6px;
+  }
+
+  .article-details h4 > a {
+    color: inherit;
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  .information span {
+    font-size: 12px;
+  }
```

Comment:
- [ ] You have this property another 9 times in your CSS. Perhaps it could be easier to create a class and add it to the HTML elements to avoid repetition here.

Reply:
Merged this where possible. Thanks